src/opm/output/eclipse/AggregateNetworkData.cpp

```diff
@@ -243,8 +246,7 @@ double nodePressure(const Opm::Schedule&               sched,
                     if (network.uptree_branch(node_name).has_value()) {
                         upt_br = network.uptree_branch(node_name).value();
                     } else {
-                        auto msg = fmt::format("Node: {} has no uptree node with fixed pressure condition,  uppermost node is: {} ", nodeName, node_name);
-                        throw std::logic_error(msg);
+                        return 0.0;  // Subtree not belonging to the network right now
```

Comment:
Have we seen the situations that part of the network only contains dangling branches, without a fix pressure node on top  so we can not update the pressures for those nodes?  It is just a question to get the scenario clear.

Reply:
Not sure if we have an actual case, but in general I think this can happen so we should cater for it.

Reply:
Let us keep the original `throw` so that we know we need to handle this specific situation.  I am worrying simply returning `0.0` will cover up the actual problem we need to tackle.

Reply:
Let us give a debug message to tell the node is not part of the active network, so we give 0 pressure. 

Then it will save time in the future if we want to do differently.

Reply:
Ok!